middleware/data_source_queries.py

```diff
@@ -132,9 +152,12 @@ def data_source_by_id_query(data_source_id="", test_query_results=[], conn={}):
     return data_source_details
 
 
-def approved_data_sources(conn) -> list:
```

Comment:
updated this function name to be more descriptive

Reply:
@maxachis sorry for the confusion, I think approved_data_sources is a clearer name

Reply:
@mbodeantor I went ahead and created a separate PR, since I went ahead and merged this one since it seemed it was considered as approved. Not sure if that was supposed to happen or if the approval process was a little wonky.